drivers/adc/ad7768-1/ad77681.c

```diff
@@ -324,6 +324,57 @@ int32_t ad77681_spi_read_adc_data(struct ad77681_dev *dev,
 	return ret;
 }
 
+/**
```

Comment:
Max 70 chars in commit messaje per line

Reply:
Done with the edits of the commit message.